### PR TITLE
feat: trade v3 ui

### PR DESCRIPTION
**Pull request by reallybeard**
## Description

- Modified trade ui to get ready for the new trade layout

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #

## Risk

This does change the layout but the logic should be the same, however this needs to be tested.

### Operations

Test trades that they work as normal. 

## Screenshots (if applicable)
Screen Shot 2023-02-13 at 4 37 38 PM
Screen Shot 2023-02-13 at 4 37 43 PM

### CODE REVIEW

File: src/components/Trade/Components/TradeQuotes/CountdownTimer.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Tooltip } from '@chakra-ui/react'
+import { CircularProgress } from 'components/CircularProgress/CircularProgress'
+
+import { useCountdown } from './useCountDown'
+
+type CountDownTimerProps = {
+  time: number
+}
+
+export const CountDownTimer: React.FC<CountDownTimerProps> = ({ time = 1 }) => {
```
Comment (0xApotheosis) [inline src/components/Trade/Components/TradeQuotes/CountdownTimer.tsx:10]: This guy isn't used anywhere?

File: src/components/Trade/Components/TradeQuotes/CountdownTimer.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  const { progress, seconds } = useCountdown(time)
+  return (
+    <Tooltip label={`something great in ${seconds} seconds`}>
```
Comment (0xApotheosis) [inline src/components/Trade/Components/TradeQuotes/CountdownTimer.tsx:13]: I'm guessing this file was some development code that needs to be removed.

File: src/components/Trade/Components/TradeQuotes/TradeQuotes.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+import { TradeQuote } from './TradeQuote'
+
+type TradeQuotesProps = {
+  isOpen?: boolean
+  isLoading?: boolean
+}
+
+export const TradeQuotes: React.FC<TradeQuotesProps> = ({ isOpen, isLoading }) => {
+  const [activeQuote, setActiveQuote] = useState('THORchain')
```
Comment (0xApotheosis) [inline src/components/Trade/Components/TradeQuotes/TradeQuotes.tsx:13]: This needs some love, obviously as it isn't wired up yet - but I'll do it in a follow-up PR 👌 

Fine for now as it's behind a feature flag and is just a template.

File: src/components/Trade/Components/TradeQuotes/useCountDown.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { bnOrZero } from '@shapeshiftoss/investor-foxy'
+import { useEffect, useState } from 'react'
+export function useCountdown(mins: number) {
```
Comment (0xApotheosis) [inline src/components/Trade/Components/TradeQuotes/useCountDown.ts:3]: Ah, this is the countdown stuff you mentioned - we might want to do this in a seperate PR, as it'll take a bit of thinking to get right.

**Review by 0xApotheosis — COMMENTED** [review]
First pass, looking solid.

Added some comments, and also have this one:

Do we really want to replace the explicit address with an account number label? I think power users would prefer to see the actual address selected (as we currently show), but that's a product question at the end of the day.

Screenshot 2023-02-08 at 9 57 14 am

File: src/components/Trade/Components/TradeQuotes/TradeQuotes.tsx (hunk 3 above)
Comment (reallybeard) [reply src/components/Trade/Components/TradeQuotes/TradeQuotes.tsx:13]: Yep didn't know how we wanted to pass around the selected quote, so just did so dumb logic to show active state.

**Comment by 0xApotheosis** [comment]
@reallybeard I'm still suss on the account labelling instead of showing the explicit address.

It's handy to see the last few characters of the account you're using as a sanity check, and it's always an inadvertent mechanism of hiding bugs.

**Comment by 0xdef1cafe** [comment]
> @reallybeard I'm still suss on the account labelling instead of showing the explicit address.
> 
> It's handy to see the last few characters of the account you're using as a sanity check, and it's always an inadvertent mechanism of hiding bugs.

big plus one on this. we could easy add a user pref to hide these if we wanted, or do the whole `0xdef1...cafe` style thing, but personally makes me quite uncomfortable

**Comment by 0xdef1cafe** [comment]
> > @reallybeard I'm still suss on the account labelling instead of showing the explicit address.
> > It's handy to see the last few characters of the account you're using as a sanity check, and it's always an inadvertent mechanism of hiding bugs.
> 
> big plus one on this. we could easy add a user pref to hide these if we wanted, or do the whole `0xdef1...cafe` style thing, but personally makes me quite uncomfortable

as discussed @reallybeard let's keep the existing way we display addresses unless there was a good reason to hide them for UI purposes?

**Comment by 0xdef1cafe** [comment]
@0xApotheosis label showing, just below

image

**Review by 0xApotheosis — APPROVED** [review]
Solid. Had a good play around and it all worked well, though we'll want ops to give this a good once over before it hits production.

File: src/components/Trade/Components/TradeQuotes/TradeQuote.tsx [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+    </Stack>
+  )
+}
+
+type TradeQuoteLoadedProps = {
+  assetId: AssetId
+  isActive?: boolean
+  isBest?: boolean
+  quoteDifference?: string
+  protocol: string
```
Comment (0xdef1cafe) [inline src/components/Trade/Components/TradeQuotes/TradeQuote.tsx:56]: would like to see this typed to the swapper enum in future PRs

**Pull request merged**